sdk/messaging/azservicebus/internal/amqpInterfaces.go

```diff
@@ -5,7 +5,9 @@ package internal
 
 import (
 	"context"
+	"time"
 
+	"github.com/Azure/azure-amqp-common-go/v3/rpc"
```

Comment:
This will go away eventually right?

Reply:
Issue tracking this is here: https://github.com/Azure/azure-sdk-for-go/issues/15646

Is there urgency around this one though? I originally thought we could do this at any time.

Reply:
No just making sure it will happen.